.github/workflows/build.yml

```diff
@@ -1,6 +1,8 @@
-name: Build (MacOS, Linux, Windows)
+name: Package (MacOS, Linux, Windows)
```

Comment:
Why this name change? I believe that build is better for a step which builds and packages the app.

Reply:
Moreover it will change the name on the badge in docs - which is bad. The badge sohuld be build, not "package OK".